bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,238 @@
+                    if question_dict["id"] not in done_question:
+                        done_question.append(question_dict["id"])
+                        break
+                q = question_dict["question"]
+                answer = question_dict["answer"]
+
+                embed = discord.Embed(colour=discord.Colour.gold())
+                embed.title = f"Question #{len(done_question)}"
+                embed.description = q
+                await ctx.send(embed=embed)
+
+            def check(m: discord.Message) -> bool:
+                ratio = fuzz.ratio(answer.lower(), m.content.lower())
+                return ratio > 80 and m.channel == ctx.channel
+            try:
+                msg = await self.bot.wait_for('message', check=check, timeout=10)
+            except Exception as e:
+                if self.game_status[ctx.channel.id] is False:
+                    break
+                if isinstance(e, asyncio.TimeoutError):
+                    if hint_no < 2:
+                        hint_no += 1
+                        if "hints" in question_dict:
+                            hints = question_dict["hints"]
+                            await ctx.send(f"**Hint #{hint_no+1}\n**{hints[hint_no]}")
+                        else:
+                            await ctx.send(f"Cmon guys, {30-hint_no*10}s left!")
+
+                    else:
+                        response = random.choice(WRONG_ANS_RESPONSE)
+                        expression = random.choice(ANNOYED_EXPRESSIONS)
+                        await ctx.send(f"{response} {expression}")
+                        await self.send_answer(ctx.channel, question_dict)
+                        await asyncio.sleep(1)
+                        hint_no = 0
+                        unanswered += 1
+                        await self.send_score(ctx.channel, player_data)
+
+            else:
+                points = 100 - 25*hint_no
+                if msg.author in player_data:
+                    player_data[msg.author] += points
+                else:
+                    player_data[msg.author] = points
+                hint_no = 0
+                unanswered = 0
+                await ctx.send(f"{msg.author.mention} got the correct answer :tada: {points} points for ya.")
+                await self.send_answer(ctx.channel, question_dict)
+                await self.send_score(ctx.channel, player_data)
+
+    @staticmethod
+    async def send_score(channel: discord.TextChannel, player_data: dict) -> None:
+        """A function which sends the score."""
+        embed = discord.Embed(colour=discord.Colour.blue())
+        embed.title = "Score Board"
+        embed.description = ""
+        for k, v in player_data.items():
+            embed.description += f"{k} : {v}\n"
+        await channel.send(embed=embed)
+
+    @staticmethod
+    async def declare_winner(channel: discord.TextChannel, player_data: dict) -> None:
+        """A function declare the winner of the quiz."""
```

Comment:
```suggestion
        """Announce the winner of the quiz in the game channel."""
```

Avoid prefixing "a function" to docstrings and try to use a more accurate description of what it's going to do, just like if someone asked you in person "what's this do if I call it?".

Reply:
Yes, sir!